lib/models/transforms.js

```diff
@@ -899,6 +899,30 @@ const Transforms = {
     })
 
     return node
+  },
+
+  /**
+   * Modify a block.
+   *
+   * @param {String} key
+   * @param {Object or String} properties
+   * @return {Node} node
+   */
+
+  modifyNode(key, properties) {
+    const node = this
+
+    node.assertDescendant(key)
```

Comment:
`assertDescendant` will actually return the match, since it has to loop the tree anyways, so we can have one less lookup here.